app/src/main/java/org/jellyfin/mobile/utils/Constants.kt

```diff
@@ -35,6 +35,7 @@ object Constants {
     const val PREF_VIDEO_PLAYER_TYPE = "pref_video_player_type"
     const val PREF_EXOPLAYER_START_LANDSCAPE_VIDEO_IN_LANDSCAPE = "pref_exoplayer_start_landscape_video_in_landscape"
     const val PREF_EXOPLAYER_ALLOW_SWIPE_GESTURES = "pref_exoplayer_allow_swipe_gestures"
+    const val PREF_EXOPLAYER_SWIPE_GESTURE_EXCLUSION_SIZE_VERTICAL = "pref_exoplayer_swipe_exclusion_size_vertical"
```

Comment:
Idk why I originally named it like that, but this is better:
```suggestion
    const val PREF_EXOPLAYER_SWIPE_GESTURE_EXCLUSION_ZONE_HEIGHT = "pref_exoplayer_swipe_exclusion_zone_height"
```

Please rename the constant/preference/string name in other places as well.